packages/api/core/src/api/init-scripts/init-npm.ts

```diff
@@ -7,12 +7,12 @@ import username from 'username';
 import { setInitialForgeConfig } from '../../util/forge-config';
 import installDepList from '../../util/install-dependencies';
 import { readRawPackageJson } from '../../util/read-package-json';
-import { version } from '../../../package.json';
 
 const d = debug('electron-forge:init:npm');
+const corePackage = fs.readJsonSync(path.resolve(__dirname, '../../../package.json'));
```

Comment:
Making it a synchronous operation, because it's still important in the initialization process.

Reply:
###### Trying to run git-cz through `yarn commit` exposes error, which is kinda weird for, because I have essential paths set: `×  fatal     Error: Could not find Git root folder.`